.github/ISSUE_TEMPLATE.md

```diff
@@ -1,16 +1,7 @@
-
-
-### What steps will reproduce the problem?
-
-### What is the expected result?
-
-### What do you get instead?
-
+| Q             | A      |
+| ------------- | ------ |
+| Is bugfix?    | ✔️/❌ |
+| New feature?  | ✔️/❌ |
+| Breaks BC?    | ✔️/❌ |
```

Comment:
This one is for pull requests. It doesn't make sense for issues.

Reply:
I reversed... 🤷🏻‍♂️🤦🏻‍♂️